consensus/src/validation/step.rs

```diff
@@ -145,7 +145,7 @@ impl<T: Operations + 'static, D: Database> ValidationStep<T, D> {
 
         for tx in candidate.txs().iter() {
             // Validate blobs
```

Comment:
I think we should do the `blob_check` here too

Reply:
It's already done by the vm execute
No need to duplicate the checks here

Here we only want to perform additional checks that not pertain to execution (due to potentially missing sidecar during sync)